common/src/main/java/com/wynntils/models/statuseffects/type/StatusEffect.java

```diff
@@ -9,15 +9,18 @@
 public class StatusEffect {
     private final StyledText fullName;
     private final StyledText name; // The name of the consumable (also used to identify it)
+    private final StyledText modifier; // The modifier of the consumable (+100, 23/3s etc.)
     private StyledText displayedTime; // The displayed time remaining. Allows for xx:xx for infinite time effects.
     private StyledText prefix; // The prefix to display before the name. Not included in identifying name.
 
-    public StatusEffect(StyledText name, StyledText displayedTime, StyledText prefix) {
+    public StatusEffect(StyledText name, StyledText modifier, StyledText displayedTime, StyledText prefix) {
         this.name = name;
         this.displayedTime = displayedTime;
         this.prefix = prefix;
-        this.fullName =
-                StyledText.concat(prefix, StyledText.fromString(" "), name, StyledText.fromString(" "), displayedTime);
+        this.modifier = modifier;
+
+        final StyledText space = StyledText.fromString(" ");
+        this.fullName = StyledText.concat(prefix, space, modifier, space, name, space, displayedTime);
```

Comment:
The newline and extracting into "space" is arguably messier, so please revert that. Thanks.

Reply:
Done, the newline was put there by the formatter I think

Reply:
Did you not push yet? I can't see this being fixed.

Reply:
Haven't pushed yet. It also appears that the formatter puts each argument on a new line now:

```java
        this.fullName = StyledText.concat(
                prefix,
                StyledText.fromString(" "),
                modifier,
                StyledText.fromString(" "),
                name,
                StyledText.fromString(" "),
                displayedTime);
```

Reply:
Yeah, thats okay.